io.github.btpf.alexandria.yml

```diff
@@ -35,10 +35,35 @@ modules:
       - -DCMAKE_BUILD_TYPE=Release
       - -DENABLE_DOCUMENTATION=OFF
       - -DENABLE_GAMEPAD=OFF
-      - -DUSE_SOUP2=ON
+      - -DUSE_SOUP2=ON # This will build webkit2gtk against libsoup 2 which is required with tauri v1
       - -DENABLE_BUBBLEWRAP_SANDBOX=OFF
     modules:
       - shared-modules/libsoup/libsoup-2.4.json
+  - name: unifdef
```

Comment:
Dependencies need to go before webkitgtk module.

Reply:
it's nested it's fine.

Reply:
Ah right, hard to see in the diff view.

Reply:
I have no idea what unifdef is for the libjpegxl needs to be build before webkit or "nested":

> `--  USE_JPEGXL                                      OFF`

Or not at all if you don't care about JPEGXL. (IE doesn't)

Reply:
I don't think I actually need JPEGXL Support, so I can remove that. 
Should I add "-DUSE_JPEGXL=OFF" to the config-opts?

Reply:
it didn't seem to care. It just couldn't find it and moved on.

Reply:
It'll fail if JEPGXL isn't found. They turned on JPEGXL by default in 2.42.0

```
-- Could NOT find JPEGXL (missing: JPEGXL_LIBRARY JPEGXL_INCLUDE_DIR) (Required is at least version "0.7.0")
CMake Error at Source/cmake/OptionsGTK.cmake:424 (message):
  libjxl is required for USE_JPEGXL
```

But yea should be fine to turn off.

unifdef is used to do some kind of post processing on headers, it's a required dependency https://github.com/WebKit/WebKit/blob/5ed23f1cad22d82c3f685c5d137eec943a864bcf/Source/cmake/OptionsGTK.cmake#L28

Reply:
this thing is if these were required the build would have failed. empirical evidence.
For example jpegxl is disabled as quoted above.

Reply:
It's enabled starting in 2.42.0, the previous build is on an older version. And yea it does fail https://buildbot.flathub.org/#/builders/6/builds/66030

Reply:
Oh I look at the wrong build. But then will 2.42 build properly for Tauri?

Let's see.

Reply:
The nesting is wrong. It needs to go inside modules I think.

Reply:
yes.

Reply:
Is this nesting correct?
https://github.com/flathub/flathub/blob/febbe4d97b4de8dc9c45c2d08d638c1b2f60a44a/io.github.btpf.alexandria.yml#L40

Reply:
Yes, it's correct now